Apps/ValidationTests/Shared/TestUtils.h

```diff
@@ -54,7 +59,7 @@ namespace Babylon
         {
             const int32_t exitCode = info[0].As<Napi::Number>().Int32Value();
 #ifdef WIN32
-            PostMessageW((HWND)_nativeWindowPtr, WM_QUIT, exitCode, 0);
+            PostMessageW((HWND)_nativeWindowPtr, WM_DESTROY, exitCode, 0);
```

Comment:
I think you probably want to use WM_CLOSE? https://stackoverflow.com/questions/3155782/what-is-the-difference-between-wm-quit-wm-close-and-wm-destroy-in-a-windows-pr